example/example.go

```diff
@@ -0,0 +1,156 @@
+package example
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/canonical/sqlair"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type Location struct {
+	ID   int    `db:"room_id"`
+	Name string `db:"name"`
+	Team string `db:"team"`
+}
+
+type Person struct {
+	Name string `db:"name"`
+	ID   int    `db:"id"`
+	Team string `db:"team"`
+}
+
+func example() {
+	sqldb, err := sql.Open("sqlite3", ":memory:")
+	if err != nil {
+		panic(err)
+	}
+
+	db := sqlair.NewDB(sqldb)
+	create := sqlair.MustPrepare(`
+	CREATE TABLE person (
+		name text,
+		id integer,
+		team text
+	);
+	CREATE TABLE location (
+		room_id integer,
+		name text,
+		team text
+	)`)
+	err = db.Query(nil, create).Run()
+	if err != nil {
+		panic(err)
+	}
+
+	insertPerson := sqlair.MustPrepare("INSERT INTO person (name, id, team) VALUES ($Person.name, $Person.id, $Person.team);", Person{})
+
+	var al = Person{"Alasatir", 1, "engineering"}
+	var ed = Person{"Ed", 2, "engineering"}
+	var marco = Person{"Marco", 3, "consulting"}
```

Comment:
I *really* think this should be fixed as Marco doesn't seem a name right for consulting lol

Reply:
hmmm, maybe `mrManino` then?

Reply:
Fine, I'll change it to engineering